src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ITypeSymbolExtensions.cs

```diff
@@ -22,7 +22,9 @@ internal static partial class ITypeSymbolExtensions
         public static ExpressionSyntax GenerateExpressionSyntax(
             this ITypeSymbol typeSymbol)
         {
-            return typeSymbol.Accept(ExpressionSyntaxGeneratorVisitor.Instance).WithAdditionalAnnotations(Simplifier.Annotation);
+            var expression = typeSymbol.Accept(ExpressionSyntaxGeneratorVisitor.Instance);
+            RoslynDebug.Assert(expression is not null);
+            return expression.WithAdditionalAnnotations(Simplifier.Annotation);
```

Comment:
what happened here that needed this?

Reply:
@CyrusNajmabadi @tmat Please hold off on reviewing the IDE portion (it's not complete yet). I'll ping once the compiler portion is reviewed. Thanks
In short, `ISerializerService.Deserialize` is missing a `?` annotation on the return type, and there are ripples from that.